waltz-ng/client/data-types/data-type-view.html

```diff
@@ -49,6 +49,25 @@
     </waltz-rated-flow-boingy-graph>
 
 
+    <!-- NON AUTH SOURCES -->
+    <waltz-section name="Non Authoritative Sources"
+                   id="non-authoritative-sources-section"
+                   icon="asterisk">
+        <waltz-non-auth-sources-list distributors="ctrl.flowDistributors"
+                                     consumers="ctrl.viewData.allConsumers"
+                                     data-data-types="ctrl.viewData.dataTypes"
+                                     ng-if="ctrl.getObjectLength(ctrl.flowDistributors) > 0">
```

Comment:
perhaps `!(ctrl.flowDistributors | isEmpty)`  (or use an `ng-show` without the negation) ?